src/structures/WidgetUser.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const Base = require('./Base');
+
+/**
+ * Represents a user on a Widget of Discord.
+ * @extends {Base}
+ */
+class WidgetUser extends Base {
+  /**
+   * @param {Client} client The instantiating client
+   * @param {Object} data The data for the user
+   */
+  constructor(client, data) {
+    super(client);
+    this._patch(data);
+  }
+
+  _patch(data) {
+    this.username = data.username;
+    this.discriminator = '0000';
```

Comment:
data.discriminator exists, let's use it

Reply:
For everybody, unfortunately, it's equal to 0000